en/ui-system/components/editor/webview.md

```diff
@@ -0,0 +1,210 @@
+# WebView Component Reference
+
+WebView is a component for displaying web pages, you could use this component to embed a mini web browser in your games. Because different platforms have different authorization, API and control methods for WebView component. And have not yet formed a unified standard, only **Web**, **iOS**, and **Android** platforms are currently supported.
+
+![webview](./webview/webview.png)
+
+Click **Add Component** at the bottom of **Properties** panel and select **WebView** from **UI Component** to add the WebView component to the node.
+
+For more information about WebView API, please refer to [WebView API](../../../api/en/classes/WebView.html) for details.
+
+## WebView Properties
+
+| Properties | Function Explanation
+|-------- | ----------- |
+| Url | A given URL to be loaded by the WebView, it should have a http or https prefix.
+| Webview Events | The webview's event callback, it will be triggered when certain webview event occurs.
+
+**Note**: In **cc.Node** of the **Webview Events** property, you should fill in a Node that hangs the user script component, and in the user script you can use the relevant WebView event according to the user's needs.
+
+## WebView Event
+
+### WebViewEvents Event
+
+| Properties |   Function Explanation
+| -------------- | ----------- |
+|Target| Node with the script component.
+|Component| Script component name.
+|Handler| Specify a callback, when the WebView is loading the web pages, or the loading is finished or there are errors occurred. The callback will be called. For more information, please refer to `Parameter of WebViewEvents`.
+| CustomEventData | The user specifies that any string is passed in as the last parameter of the event callback. |
+
+For more information, please refer to [Component.EventHandler Class](../../../api/en/classes/Component.EventHandler.html).
```

Comment:
For more information, please refer to [Component.EventHandler Class](../../../api/en/classes/Component.EventHandler.html).
-->
For more information, please refer to the [Component.EventHandler Class](../../../api/en/classes/Component.EventHandler.html) documentation.